PWGDQ/Core/HistogramsLibrary.h

```diff
@@ -311,4 +311,42 @@ void o2::aod::dqhistograms::DefineHistograms(HistogramManager* hm, const char* h
     hm->AddHistogram(histClass, "DeltaEta_DeltaPhi", "", false, 20, -2.0, 2.0, VarManager::kDeltaEta, 50, -8.0, 8.0, VarManager::kDeltaPhi);
     hm->AddHistogram(histClass, "DeltaEta_DeltaPhiSym", "", false, 20, -2.0, 2.0, VarManager::kDeltaEta, 50, -8.0, 8.0, VarManager::kDeltaPhiSym);
   }
+
+  if (groupStr.Contains("postcalib")) {
```

Comment:
You do not need to make a new group in the library. Just make the "postcalib" as a subgroup of the "track" group. The reason for this is that it should be more or less clear from the group name, where these histograms are being filled. So for example, histograms in "track" are filled in the track loops, etc.
Then i think you do not need the "electron", pion and proton subgroups. When we postcalibrate, we normally do all the species together, so when you switch "postcalib" on, you should include all those histograms.

Reply:
HI, Ionut
Now, I moved the "postcalib" to the track histogram as you suggested, 
Cheers
Xiaozhi